plugins/thunderbolt/fu-self-test.c

```diff
@@ -64,14 +64,15 @@ udev_mock_add_domain (UMockdevTestbed *bed, int id)
 
 
 static gchar *
-udev_mock_add_nvme_nonactive (UMockdevTestbed *bed,
-			      const char      *parent,
-			      int              id)
+udev_mock_add_nvme (UMockdevTestbed *bed,
```

Comment:
Shouldn't this be udev_mock_add_nvme**m**?  Simple typo I think where it's been used.

Reply:
Correct. nvme is a something else. Thanks!